src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StagedPaymentStep/partials/MilestoneReleaseModal/consts.ts

```diff
@@ -8,18 +8,22 @@ export const getMilestoneReleaseDecisionMethodDescriptions = (
 ) => ({
   [DecisionMethod.Permissions]: formatText(
     {
-      id: 'fundingModal.permissionsDescription',
+      id: 'milestoneModal.permissionsDescription',
     },
     {
       permission: userRole,
     },
   ),
   [DecisionMethod.Reputation]: formatText({
-    id: 'fundingModal.reputationDescription',
+    id: 'milestoneModal.reputationDescription',
   }),
   [DecisionMethod.PaymentCreator]: formatText({
-    id: 'releaseModal.paymentCreatorDescription',
+    id: 'milestoneModal.paymentCreatorDescription',
   }),
+  // @TODO: Add multi-sig
+  // [DecisionMethod.PaymentCreator]: formatText({
```

Comment:
Here we should use `DecisionMethod.MultiSig` - but either way as it's disabled for the moment you can leave it as it is